### PR TITLE
yplan: add page

### PR DESCRIPTION
**Pull request by DEBADRIBASAK**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Closes https://github.com/tldr-pages/tldr/issues/8860

### CODE REVIEW

File: pages/linux/yplan.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# yplan
+
+> A Perl prgram that generates LaTex code to print a vertical-type size-monthly daily planner for any chosen year.
```
Comment (reinhart1010) [inline pages/linux/yplan.md:3]:
```suggestion
> Generate LaTeX code for a two-page vertical daily planner for any chosen year.
> The output can be converted or printed using conversion tools such as `pandoc`, `pdflatex`, or `xetex`.
```

Simpler than (alongside the typo):

> Prints two six-monthly vertical-type daily planner (i.e., months along the top, days downwards), with each 6-month period fitting onto a single A4 (or US letter) sheet.

File: pages/linux/yplan.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://www.ctan.org/tex-archive/macros/latex/contrib/yplan>.
+
+- Create a daily planner with specified language, lettercase (uppercase or lowercase) and year:
+
+`yplan {{language}} {{lettercase}} {{year}} > {{file.tex}}`
```
Comment (reinhart1010) [inline pages/linux/yplan.md:8]:
```suggestion
`yplan {{language}} {{lettercase}} {{year}} > {{path/to/file.tex}}`
```

You're using shell redirection so it applies to non-current directory as well.

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for contributing! Here are some of my notes to review.

**Review by kbdharun — APPROVED** [review]
LGTM, after @reinhart1010's suggestions.

**Comment by gutjuri** [comment]
closes #8860

File: pages/linux/yplan.md (hunk 2 above)
Comment (DEBADRIBASAK) [reply pages/linux/yplan.md]: Done

File: pages/linux/yplan.md (hunk 1 above)
Comment (DEBADRIBASAK) [reply pages/linux/yplan.md]: Done.

**Review by kbdharun — APPROVED** [review]

**Review by reinhart1010 — APPROVED** [review]
Thabks for your revision!

**Pull request merged**